tests/xla_bridge_test.py

```diff
@@ -56,6 +64,15 @@ def test_local_devices(self):
     with self.assertRaisesRegex(RuntimeError, "Unknown backend foo"):
       xb.local_devices(backend="foo")
 
+  @mock.patch('jax.lib.xla_client.make_tpu_client', side_effect=mock_tpu_client)
```

Comment:
I think this is the first use of mocks in jax, so cool! :D

Reply:
Woohoo!